src/vs/workbench/contrib/debug/browser/breakpointEditorContribution.ts

```diff
@@ -236,6 +237,22 @@ export class BreakpointEditorContribution implements IBreakpointEditorContributi
 	}
 
 	private registerListeners(): void {
+		this.toDispose.push(
+			GutterActionsRegistry.registerGutterActionsGenerator(({ lineNumber, editor }, result) => {
```

Comment:
This is a per-editor contribution, but the registry is a global thing, so I think this should get registered somewhere else